### DISCUSSION TITLE
Next Steps

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#107] [General]
The next step should be the creation of a Stable version.

To have a Stable version (in my understanding) an implementation of the CFS API must be carried on providing back feedbacks (bugs or new requirements) for us to implement (if agreed).

Next steps toward the creation of a Stable version are:

- Fix bugs
- Implement new features if the new requirements are defined and agreed
- Keep the CFS API aligned with Commonalities

**Comment by FabrizioMoggio** [comment]
Are there plans for a CFS API implementation by any Partner?

**Reply by bigludo7** [reply]
Not in short term on my side @FabrizioMoggio 
This is an API that we have in the horizon and backlog but not with top priority ... as of now.

**Comment by FabrizioMoggio** [comment]
TIM has shifted the scheduling for the CFS API implementation to 2025

**Comment by StefanoFalsetto-CKHIOD** [comment]
CKH would only have considered implementing at v1.0 but we can do it with v0.2 by the end of this year on the following assumption:

1.  We can get GSMA certified using 2-legged Client Credentials authentication mechanism 
2. The API is sufficiently stable (i.e., if all goes well, the future v1.0 will just be a rename from v0.2)

**Reply by FabrizioMoggio** [reply]
@StefanoFalsetto-CKHIOD in this discussion we defined the API as 3-legged. https://github.com/camaraproject/CallForwardingSignal/discussions/7

Do you mean that you will try to engage GSMA to see if you can get their certification on a 2-legged implementation?

**Reply by StefanoFalsetto-CKHIOD** [reply]
Yes, we are waiting for feedbacks from them. I'll keep you updated.

**Comment by StefanoFalsetto-CKHIOD** [comment]
Hi All,
CKH confirms the implementation of version 0.2.0 of this API using Client Credentials authentication, by the end of the year.

**Comment by FabrizioMoggio** [comment]
Stefano this is good news, if you will provide the group with feedbacks on the API we can improve it and move toward a stable version for the next CAMARA Meta-release.

**Comment by FabrizioMoggio** [comment]
what discussed here is currently ongoing
